yarn.lock

```diff
@@ -247,6 +247,21 @@
     exec-sh "^0.3.2"
     minimist "^1.2.0"
 
+"@homebound/pg-structure@^7.2.4":
```

Comment:
Hm, I don't read `.lock` file diffs very often, but should we have red lines that show the non-homebound pg-structure isn't pulled in anymore?

Reply:
yeah weird. good catch